src/native/managed/cdacreader/tests/MockDescriptors/MockDescriptors.MethodDescriptors.cs

```diff
@@ -44,7 +44,7 @@ public class MethodDescriptors
         internal readonly Loader LoaderBuilder;
 
         internal Dictionary<DataType, Target.TypeInfo> Types { get; }
-        internal (string Name, ulong Value, string? Type)[] Globals { get; }
+        internal (string Name, ulong Value)[] Globals { get; }
```

Comment:
Why is the `string? Type` no longer needed for globals when testing? Looking at how they are used in the contracts, it seems the contracts are assumed to already know the type and reference them by name.

It looks like the `ContractDescriptor` JSON encodes the global type information and it is returned when reading a global. I don't see any contracts that use this, but it is part of the available information, is this needed? If so, do we need to replicate it during testing?